packages/web/src/pages/search-page/search-results/AllResults.tsx

```diff
@@ -73,7 +75,9 @@ export const AllResults = () => {
             count={12}
             viewLayout='grid'
             category={SearchKind.ALL}
-            queryData={{ ...queryData, data: queryData?.data?.tracks }}
+            isFetching={isLoading}
```

Comment:
All the other components in this file are passing `data` through, but I don't see it here?